cloudflare.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"io"
 	"io/ioutil"
+	"log"
```

Comment:
We definitely shouldn't log in the library (it breaks expectations around how applications are handling their own logs). If you need debug logs, we should bubble that up in the errors, or you can provide a custom `http.Client` that logs the request/response separately.